tests/mcuboot/Makefile

```diff
@@ -0,0 +1,15 @@
+APPLICATION = hello_mcuboot
+
+BOARD ?= nrf52dk
+
+include ../Makefile.tests_common
+
+BOARD_WHITELIST := nrf52dk
+
+export IMAGE_VERSION = 1.1.1+1
```

Comment:
Oh, those variables are CPU dependent, since the alignment can change according to the specific alignment of the internal flash (mostly is 4, but for some CPUs is 8). IMG_HDR is also CPU specific, so maybe I can move these two exports to the Makefile.include of the CPU.